timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -166,28 +166,6 @@ public void multipleClientsCanUseSharedThreads() throws Exception {
         assertThat(lockAndUnlockAndCountExceptions(lockServiceList, MAX_CONCURRENT_LOCK_REQUESTS / 3)).isEqualTo(0);
     }
 
-    @Test
-    public void throwsOnSingleClientRequestingSameLockTooManyTimes() throws Exception {
-        List<LockService> lockServiceList = ImmutableList.of(getLockService(CLIENT_1));
-
-        int exceedingRequests = 10;
-        int maxRequestsForOneClient = MAX_CONCURRENT_LOCK_REQUESTS;
-
-        assertThat(lockAndUnlockAndCountExceptions(lockServiceList, exceedingRequests + maxRequestsForOneClient))
-                .isEqualTo(exceedingRequests);
-    }
-
-    @Test
-    public void throwsOnTwoClientsRequestingSameLockTooManyTimes() throws Exception {
-        List<LockService> lockServiceList = ImmutableList.of(
-                getLockService(CLIENT_1), getLockService(CLIENT_2));
-        int exceedingRequests = 10;
-        int requestsPerClient = (MAX_CONCURRENT_LOCK_REQUESTS + exceedingRequests) / 2;
-
-        assertThat(lockAndUnlockAndCountExceptions(lockServiceList, requestsPerClient))
```

Comment:
is this still used after you've gotten rid of these invocations?

Reply:
It was still used by the happy case test, though I've removed this and removed those as well as they don't really make sense after CJR.